input/input.go

```diff
@@ -72,7 +103,7 @@ func expand(pattern string) ([]string, error) {
 						return err
 					}
 
-					hits = append(hits, path)
+					hits = append(hits, strings.Replace(path, `//`, `/`, -1))
```

Comment:
I think you can use `filepath.Clean()` here...

Reply:
oh noice!